src/MCPClient/lib/clientScripts/archivematicaCreateMETS2.py

```diff
@@ -255,16 +267,15 @@ def createDublincoreDMDSecFromDBData(type, id, baseDirectoryPath):
                     parser = etree.XMLParser(remove_blank_text=True)
                     dtree = etree.parse(dcXMLFile, parser)
                     dc = dtree.getroot()
+                    break
                 except Exception as inst:
                     print >>sys.stderr, "error parsing file:", dcXMLFile
                     print >>sys.stderr, type(inst)     # the exception instance
                     print >>sys.stderr, inst.args
                     traceback.print_exc(file=sys.stdout)
                     sharedVariablesAcrossModules.globalErrorCount += 1
-                    return None
-            else:
-                return None
-
+        else:  # break not called, no DC successfully parsed
```

Comment:
There can only be one `dublincore.xml`? If there's more than one transfer, how do we determine which is used? The one with the alphabetically lowest UUID?